masterlist.yaml

```diff
@@ -7354,23 +7368,20 @@ plugins:
     tag:
       - Graphics
       - NoMerge
+
+  - name: 'MaleBodyReplacerV\d\.esp'
+    tag: [ Body-M ] # R.Body-M
   - name: 'MaleBodyReplacerV3.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/8295' ]
-    tag:
-      - Body-M
-      - Body-Size-M
+    tag: [ Eyes ] # R.Eyes
   - name: 'MaleBodyReplacerV4.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/14942' ]
-    tag:
-      - Body-M
-      - Body-Size-M
+    tag: [ Eyes ] # R.Eyes
   - name: 'MaleBodyReplacerV5.esp'
     url:
       - 'https://www.nexusmods.com/oblivion/mods/33669'
       - 'https://www.nexusmods.com/oblivion/mods/40532'
-    tag:
-      - Body-M
-      - Body-Size-M
+
```

Comment:
@Infernio Since this entry is just locations, it should be removed. However, it would be best if these tags were added in a separate PR anyway.